src/usr/local/www/includes/functions.inc.php

```diff
@@ -320,7 +320,7 @@ function crypto_accel_get_algs($crypto) {
 		$algs_str .= $alg;
 	}
 
-	return ($algs_str);
+	return (!empty($algs_str) ? $algs_str : "Inactive");
```

Comment:
This should be `gettext("Inactive")` so the string gets localized.

Reply:
Thanks @jim-p - I'll remember that next time. I pushed the update.